VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/store/effects/conference.effects.ts

```diff
@@ -84,10 +87,32 @@ export class ConferenceEffects {
         )
     );
 
+    participantDisconnect$ = createEffect(
+        () =>
+            this.actions$.pipe(
+                ofType(ConferenceActions.updateParticipantStatus),
+                filter(action => action.status === ParticipantStatus.Disconnected),
+                concatLatestFrom(() => [this.store.select(ConferenceSelectors.getLoggedInParticipant)]),
+                switchMap(([action, participant]) => {
+                    if (participant.id !== action.participantId) {
+                        return of();
+                    }
+                    const callTag = participant?.pexipInfo?.callTag ?? this.videoCallService.pexipAPI.call_tag;
```

Comment:
looks good!
did have quick question, why use callTag over the conferenceId that is already on the event? and used as key for storing the data such as isMuted etc already? Ive probably missed something

Reply:
So I discovered that the call tag isn't set until we get a callback which first arrives when you get into the hearing room. To get the call tag from the waiting room you need to check the video call service